### ISSUE TITLE
Vulnerability in qs < 6.3.2

### ISSUE THREAD

**Issue by alchin** [BrowserSync/browser-sync#1666]
### Issue details

Vulnerability in dependent qs package for versions < 6.3.2

### Steps to reproduce/test case

Install latest browser-sync package

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [2.26.3]
- Node        [8.11.3]
- Npm         [5.6.0 ]

### Affected platforms

- [ ] linux
- [x] windows
- [ ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [ ] Gulp
- [ ] Grunt
- [ ] CLI

### If CLI, please paste the entire command below

{cli command here}

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

{Browsersync init code here}

**Comment by rbiggs** [comment]
Same problem. Please address this vulnerability ASAP.

**Comment by SailingSteve** [comment]
Would appreciate an update

**Comment by rbiggs** [comment]
This is what I got when I installed Browsersync with the latest version:

```bash
$: npm i -D browser-sync
+ browser-sync@2.26.3
updated 1 package and audited 13607 packages in 13.283s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
$: npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ braces                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.3.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ browser-sync [dev]                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ browser-sync > micromatch > braces                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/786                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 13607 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

Apparently Browsersync uses braces or has a dependency that does. It's braces that has the vulnerability.

**Comment by TigerC10** [comment]
`npm list braces`
```
├─┬ browser-sync@2.26.3
│ ├─┬ chokidar@2.1.4
│ │ ├─┬ anymatch@2.0.0
│ │ │ └─┬ micromatch@3.1.10
│ │ │   └── braces@2.3.2  deduped
│ │ ├── braces@2.3.2
│ │ └─┬ readdirp@2.2.1
│ │   └─┬ micromatch@3.1.10
│ │     └── braces@2.3.2  deduped
│ └─┬ micromatch@2.3.11
│   └── braces@1.8.5
└─┬ webpack@4.29.6
  └─┬ micromatch@3.1.10
    └── braces@2.3.2```

Looks like it's the `micromatch` dependency that needs to be updated.

**Comment by shakyShane** [comment]
fixed in 2.26.4 :)
